api/v1alpha2/machine_types.go

```diff
@@ -37,9 +37,11 @@ const (
 
 // MachineSpec defines the desired state of Machine
 type MachineSpec struct {
-	// ObjectMeta will autopopulate the Node created. Use this to
+	// DEPRECATED: ObjectMeta will autopopulate the Node created. Use this to
```

Comment:
Let's not leave the text in here about autopopulating stuff. It's incorrect.

Reply:
ptal